src/yandex_cloud_ml_sdk/_files/domain.py

```diff
@@ -18,6 +18,9 @@
 
 
 class BaseFiles(BaseDomain, Generic[FileTypeT]):
+    """Base class for file operations in Yandex Cloud.
```

Comment:
I dont like this "Base class" wording will get into Files and Async files documentation because of the `@doc_from`.

Maybe something like "Files domain, which contains API for working with files: <link on the doc>"

Maybe also it is good to note that Files is part of Assistants API (and it is only place you coud use it)